src/main/kotlin/com/terraformation/backend/customer/model/IndividualUser.kt

```diff
@@ -734,6 +682,13 @@ data class IndividualUser(
 
   override fun canUploadPhoto(accessionId: AccessionId) = canReadAccession(accessionId)
 
+  // When adding new permissions, put them in alphabetical order.
```

Comment:
I had this at the bottom of the class so that people wouldn't be tempted to just put new permission methods at the bottom. I think it makes sense to either keep it there or remove it; its intended audience probably won't notice it in the middle like this.

Reply:
Yeah, adding it to `CONVENTIONS.md` seems reasonable. I suppose once we've figured out what the rule actually is, it'd be nice to go through the code and make sure we're following it consistently. (That may be the kind of thing an AI coding assistant would be good at.)